pkg/postage/stampissuer_test.go

```diff
@@ -40,3 +40,16 @@ func newTestStampIssuer(t *testing.T, block uint64) *postage.StampIssuer {
 	}
 	return postage.NewStampIssuer("label", "keyID", id, big.NewInt(3), 16, 8, block, true)
 }
+
+func TestGetUnexpiredStampIssuer(t *testing.T) {
+	t.Helper()
+	id := make([]byte, 32)
+	_, err := io.ReadFull(crand.Reader, id)
```

Comment:
There is already a testing function to create a new Batch. It should be in postage testing pkg.